File: solutions/python/isogram/1/isogram.py
def is_isogram(string):
    if string=='':
        return True
    
    string=string.lower()
    appears=[False]*26

    for char in string:
        index=ord(char)-ord('a')
        if index<0 or index>26:
            continue

        if appears[index] is True:
            return False
        else: 
            appears[index]=True

    return True

File: solutions/dart/isogram/1/lib/isogram.dart
class Isogram {
  bool isIsogram(String word) {
    if (word == '') {
      return true;
    }

    final occurence = List.filled(26, false);
    
    for (var c in word.split('')) {
      if (c == '-'||c==' ') continue;

      int index = c.toLowerCase().codeUnitAt(0) - 'a'.codeUnitAt(0);
      if (occurence[index] == true) {
        return false;
      } else {
        occurence[index] = true;
      }
    }

    return true;
  }
}
